.github/workflows/ci.yml

```diff
@@ -20,9 +20,8 @@ jobs:
           java-version: ${{ matrix.java }}
       - run: 
           # for GitOps tests
-          git config --global user.email "scalafmt@scalameta.org"
-          git config --global user.name "scalafmt"
-          sbt ci-test
+          git config --global user.email "scalafmt@scalameta.org" && git config --global user.name "scalafmt"
```

Comment:
Would it work instead:
```
GIT_AUTHOR_NAME=... GIT_AUTHOR_EMAIL=... sbt ci-test
```

Reply:
looks like it didn't. the previous solution is good.

Reply:
Environment variables don’t transfer between runs unless you explicitly export them by printing some special syntax.